hw/ip/otbn/otbn.core

```diff
@@ -20,6 +20,11 @@ filesets:
       - rtl/otbn_core.sv
       - rtl/otbn.sv
     file_type: systemVerilogSource
+  files_model:
+    files:
+      - rtl/dv/dpi/otbn_model.cc: { file_type: cppSource }
+      - rtl/dv/otbn_core_model.sv
```

Comment:
These files are missing from the PR?

And can you put both files into `rtl/dv/model` to have a separate directory for them, away from the UVM DV?